src/main/java/org/spongepowered/common/data/manipulator/mutable/entity/SpongeMovementSpeedData.java

```diff
@@ -84,8 +84,8 @@ protected void registerGettersAndSetters() {
     @Override
     public MutableBoundedValue<Double> walkSpeed() {
         return SpongeValueBuilder.boundedBuilder(Keys.WALKING_SPEED)
-            .defaultValue(0.1D)
-            .minimum(Double.MIN_VALUE)
+            .defaultValue(0.699999988079071D) // Why not 0.7D? Ask Mojang
```

Comment:
@gabizou - The 0.7D was got from the `SharedMonsterAttributes` - as far as I can tell this is what's used in vanilla as the default movement speed, with some mobs overriding that.